monitoring/views/bigquery_usage.view.lkml

```diff
@@ -0,0 +1,39 @@
+include: "//looker-hub/monitoring/views/bigquery_usage.view.lkml"
+
+view: +bigquery_usage {
+
+  dimension: job_id {
+    primary_key: yes
+    sql: job_id ;;
```

Comment:
I think this might not be needed since the field name is the same as the looker dimension
```suggestion
```

Reply:
I need to set the primary key to do distinct count in measures